persist/fs/commitlog/commit_log.go

```diff
@@ -376,9 +396,7 @@ func (l *commitLog) WriteBehind(
 	unit xtime.Unit,
 	annotation ts.Annotation,
 ) error {
-	l.RLock()
-
-	if l.closed {
+	if l.RLock(); l.closed {
```

Comment:
Here too, seems `l.RLock()` is not pertinent to the condition, seems it would belong better to the lexical scope above.